rusqlite_migration/src/lib.rs

```diff
@@ -141,6 +139,8 @@ use std::{
     ptr::addr_of,
 };
 
+const NO_PARAMS: &[&dyn rusqlite::ToSql] = &[];
+
```

Comment:
```suggestion
```
I would suggest using `[]` where needed instead, since we are updating rusqlite in this PR anyway.